app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt

```diff
@@ -130,7 +126,11 @@ class SignUpFragment : Fragment() {
     }
 
     private fun redirectToMain() {
-        findNavController(rootView).popBackStack()
+        val intent = Intent(activity, MainActivity::class.java)
+        intent.putExtra("snackbar_message", getString(R.string.logged_in_automatically))
+        startActivity(intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP))
+        activity?.overridePendingTransition(R.anim.slide_in_left, R.anim.slide_out_right)
+        activity?.finish()
```

Comment:
Revert please. We are now single activity application

Reply:
Pls suggest alternate way I'm new to navigation architecture

Reply:
Read about it